packages/editor/src/components/block-inspector/style.scss

```diff
@@ -48,5 +48,5 @@
 }
 
 .editor-block-inspector__styles .editor-block-styles__item {
-	width: 100%;
+	width: 47%;
```

Comment:
I think you can just remove this style entirely as 50% is the default style applied.

Reply:
50% doesn't look very good.

Reply:
It's actually `width: calc(50% - #{ $item-spacing / 2 });` which I guess is very close to `47%`

Reply:
Feel free to address this and let's merge :)